README.md

```diff
@@ -13,7 +13,7 @@
 
 ## Description
 
-Collection is a functional utility library for PHP greater than 7.4.
+Collection is a functional utility library for PHP greater than 7.4, including PHP 8.0.
```

Comment:
I think it's good to mention this to reassure users that they can use it in PHP 8

Reply:
Sure it's good